objectRecognitionSource/README.md

```diff
@@ -673,6 +679,9 @@ OpenCV will also display errors that look like `warning: Error opening file
 (/build/opencv/modules/videoio/src/cap_ffmpeg_impl.hpp:857)`. These are expected, as the tests need to ensure correct
 behavior when the file cannot be found.
 
+Some sets of tests will create a Source named "TestObjectRecognitionSource" in the VANTIQ Namespace associated with the 
```

Comment:
Would it be worth it to provide an override -- allow a gradle property to set this -- otherwise, we'll use this name?

Might be good to name it something less likely to exist -- UnlikelyToExistTestObjectRecognizer or something like that, as _TestObjectRecognizer_ might be the first thing anyone tries.

Reply:
It might enough to add a prefix, e.g. VV__TestObjectRecognitionSource